### PR TITLE
1.12.0.28 Server Support

### PR DESCRIPTION
**Pull request by ChristopherHX**
Added new / changed 1.12 and 1.13 Symbols

### CODE REVIEW

File: src/minecraft/ResourcePackStack.h [hunk 1]
```diff
@@ -8,7 +8,7 @@ class PackSettings;
 
 struct PackInstance {
 
-    char filler[0x7C];
+    char filler[0x8C];
```
Comment (ChristopherHX) [inline src/minecraft/ResourcePackStack.h]: The real size of PackInstance is unknown, but must be greater equal than 0x9C.
No crash with ~~0x8C~~ ( Impacts ServerProperties silently ) 0x9C, no more undefined behavior noticed.
Fixed shutdown crash 1.12 server

**Comment by 67au** [comment]
Will this PR support ARM ver. ?

**Comment by ChristopherHX** [comment]
> Will this PR support ARM ver. ?


This PR is currently almost armhf compatible, only FilePathManager::getWorld() seems to be broken on my arm raspberry pi 2.
Avoided now in my Server PR works well on armhf tested
My current working armhf client for armhf 1.12.1.1 (Raspberry Pi 2 Tested) https://github.com/ChristopherHX/mcpelauncher-manifest/tree/1.12.x_armhf
Only Symbols with generated armhf symbolwrapper, can be generated with python on this PR, https://github.com/ChristopherHX/minecraft-symbols/tree/1.12.x_armhf

File: src/minecraft/FilePathManager.h [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
+    FilePathManager(Legacy::Pre_1_13::Core::Path const&, bool);
 
-    mcpe::string getRootPath() const;
+    const mcpe::string &getRootPath() const;
 
-    mcpe::string getUserDataPath() const;
+    const mcpe::string &getUserDataPath() const;
 
-    mcpe::string getWorldsPath() const;
+    const mcpe::string &getWorldsPath() const;
```
Comment (ChristopherHX) [inline src/minecraft/FilePathManager.h:21]: Broken on armhf unknown error, perhabs Symbol exists.
Now avoid calling getWorldsPath in server impl

**Pull request merged**